TestModels/SimpleTypes/SimpleBlob/runtimes/rust/src/conversions/get_blob/_get_blob_input.rs

```diff
@@ -26,9 +26,10 @@ pub fn from_dafny(
         dafny_value.value().as_ref(),
         ::simple_blob_dafny::_Wrappers_Compile::Option::Some { .. }
     ) {
-        Some(::std::rc::Rc::unwrap_or_clone(
-            dafny_value.value().Extract().to_array(),
-        ))
+        Some(
+            ::std::rc::Rc::try_unwrap(dafny_value.value().Extract().to_array())
```

Comment:
This looks like the right approach AFAICT. I'm not 100% sure it's the best we can do in the limit, but I am sure that it's much easier to optimize this kind of thing once we're actually generating the code, so it should be very quick to fix if we do realize we can do better latter. E.g. we may want to add this conversion to the Dafny Rust runtime.